cohort-evaluator-spark/src/test/java/com/ibm/cohort/cql/spark/SparkCqlEvaluatorTest.java

```diff
@@ -143,6 +143,30 @@ public void testAllTypesEvaluationSuccess() throws Exception {
         validateOutputCountsAndColumns(dFile.toURI().toString(), new HashSet<>(Arrays.asList("id", "MeasureD|cohort")), 567);
     }
 
+    @Test
+    public void testAnyColumnEvaluation() throws Exception {
+        File inputDir = new File("src/test/resources/alltypes/");
+        File outputDir = new File("target/output/any-column/");
+
+        File aFile = new File(outputDir, "A_cohort");
+
+        String [] args = new String[] {
+            "-d", "src/test/resources/any-column/metadata/context-definitions.json",
+            "-j", "src/test/resources/any-column/metadata/cql-jobs.json",
+            "-m", "src/test/resources/alltypes/modelinfo/alltypes-modelinfo-1.0.0.xml",
+            "-c", "src/test/resources/any-column/cql",
+            "--input-format", "parquet",
+            "-i", "A=" + new File(inputDir, "testdata/test-A.parquet").toURI(),
+            "-o", "A=" + aFile.toURI(),
+            "-n", "10",
+            "--overwrite-output-for-contexts"
+        };
+
+        SparkCqlEvaluator.main(args);
+
+        validateOutputCountsAndColumns(aFile.toURI().toString(), new HashSet<>(Arrays.asList("id_col", "MeasureAnyColumn|cohort")), 572);
```

Comment:
I like that there is this test to make sure that the external function provider is hooked in properly and that the evaluations succeed for the AnyColumn function. If you wanted bonus testing points, you could mimic `testValidateOutput()` and use a smaller set of handcrafted data to run and then verify the exact results.

You already have unit tests in `AnyColumnTest`, so it's not the end of the world to just verify the execution succeeded.

Reply:
addressed in 89ad0cf